### PR TITLE
[SYCLomatic] Fix bug of memcpy_parameter used to migrate Drive APIs.

### PR DESCRIPTION
**Pull request by tangjj11**
Signed-off-by: Tang, Jiajun jiajun.tang@intel.com

### CODE REVIEW

**Review by ziranzha — APPROVED** [review]

File: clang/runtime/dpct-rt/include/dpct/memory.hpp [hunk 1]
```diff
@@ -107,6 +113,7 @@ struct memcpy_parameter {
   struct data_wrapper {
     pitched_data pitched{};
     sycl::id<3> pos{};
+    size_t pos_x_byte{0};
```
Comment (zhimingwang36) [inline clang/runtime/dpct-rt/include/dpct/memory.hpp]:
```suggestion
    size_t pos_x_in_bytes{0};
```

File: clang/runtime/dpct-rt/include/dpct/memory.hpp [hunk 2]
```diff
@@ -116,6 +123,7 @@ struct memcpy_parameter {
   data_wrapper from{};
   data_wrapper to{};
   sycl::range<3> size{};
+  size_t size_x_byte{0};
```
Comment (zhimingwang36) [inline clang/runtime/dpct-rt/include/dpct/memory.hpp]:
```suggestion
  size_t size_x_in_bytes{0};
```

File: clang/runtime/dpct-rt/include/dpct/memory.hpp (hunk 1 above)
Comment (tangjj11) [reply clang/runtime/dpct-rt/include/dpct/memory.hpp]: done

File: clang/runtime/dpct-rt/include/dpct/memory.hpp (hunk 2 above)
Comment (tangjj11) [reply clang/runtime/dpct-rt/include/dpct/memory.hpp]: done

**Review by zhimingwang36 — APPROVED** [review]

**Pull request merged**